closed/src/java.base/share/classes/java/io/ClassCache.java

```diff
@@ -209,25 +209,24 @@ private Object createEntry(CacheKey key) {
         }
 
         public Thread run() {
-            return new Reaper(cache, queue);
+            Reaper reaper = new Reaper(cache, queue);
+            return com.ibm.oti.vm.VM.getVMLangAccess().createThread(reaper, "ClassCache Reaper", true, false, true, null);
```

Comment:
This needs a doPrivileged()

Reply:
Hmm, actually maybe only for jdk8

Reply:
Please try with a SecurityManager installed

Reply:
There is a `doPrivileged()` from caller.
https://github.com/ibmruntimes/openj9-openjdk-jdk11/blob/771d7cb5fb5ee8d83a20bfc5646b3270f55b69af/closed/src/java.base/share/classes/java/io/ClassCache.java#L67-L68

Reply:
In addition, run a small program with `new ObjectInputStream(new FileInputStream("test.txt"))` with security manager enabled, no `AccessControlException` thrown (only requires access to the file being tested).
The code snippet should create `ClassCache` in question. 

P.S. the original Renaissance Suite requires more permissions to proceed hence use the small app above instead.